src/python_testing/TC_PAVSTI_1_2.py

```diff
@@ -0,0 +1,323 @@
+            )
+            isValidIngestMethod = (
+                format.ingestMethod == pushavCluster.Enums.IngestMethodsEnum.kCMAFIngest
+            )
+            asserts.assert_true(
+                (validContainerformat and isValidIngestMethod),
+                "(ContainerFormat and IngestMethod) must be defined values",
+            )
+
+        self.step(3)
+        allocatedVideoStreams = await self.read_single_attribute_check_success(
+            endpoint=endpoint,
+            cluster=avsmCluster,
+            attribute=avsmAttr.AllocatedVideoStreams,
+        )
+        logger.info(f"Rx'd AllocatedVideoStreams: {allocatedVideoStreams}")
+        asserts.assert_true(
+            len(allocatedVideoStreams) != 0, "AllocatedVideoStreams must not be empty"
+        )
+
+        allocatedVideoStream = allocatedVideoStreams[0]
+        videoStreamId = allocatedVideoStream.videoStreamID
+
+        self.step(4)
+        allocatedAudioStreams = await self.read_single_attribute_check_success(
+            endpoint=endpoint,
+            cluster=avsmCluster,
+            attribute=avsmAttr.AllocatedAudioStreams,
+        )
+        logger.info(f"Rx'd AllocatedAudioStreams: {allocatedAudioStreams}")
+        asserts.assert_true(
+            len(allocatedAudioStreams) != 0, "AllocatedAudioStreams must not be empty"
+        )
+        allocatedAudioStream = allocatedAudioStreams[0]
+        audioStreamId = allocatedAudioStream.audioStreamID
+
+        self.step(5)
+        if self.pics_guard(self.check_pics("AVSM.S.F0003")):
+            aFeatureMap = await self.read_single_attribute_check_success(
+                endpoint=endpoint, cluster=avsmCluster, attribute=avsmAttr.FeatureMap
+            )
+            privacySupport = (aFeatureMap & avsmCluster.Bitmaps.Feature.kPrivacy) > 0
+            asserts.assert_true(privacySupport, "Privacy Feature is not supported.")
+
+            result = await self.write_single_attribute(
+                avsmAttr.SoftRecordingPrivacyModeEnabled(True), endpoint_id=endpoint
+            )
+            asserts.assert_equal(
+                result,
+                Status.Success,
+                "Error when trying to write SoftRecordingPrivacyModeEnabled",
+            )
+
+        self.step(6)
+        if self.pics_guard(self.check_pics("AVSM.S.F0003")):
+            result = await self.write_single_attribute(
+                avsmAttr.SoftLivestreamPrivacyModeEnabled(True), endpoint_id=endpoint
+            )
+            asserts.assert_equal(
+                result,
+                Status.Success,
+                "Error when trying to write SoftLivestreamPrivacyModeEnabled",
+            )
+
+        self.step(7)
+        allocatePushTransportResponse = await self.send_single_cmd(
+            cmd=pushavCluster.Commands.AllocatePushTransport(
+                transportOptions={
+                    "streamUsage": Globals.Enums.StreamUsageEnum.kInternal,
```

Comment:
Same comment as earlier, the AVSM methods pick the first usage in the supported set, which I think is "kLiveView".  Do we want to explicity setup a recording stream instead?

Reply:
Thank you for your review. We had fixed this similarly by explicitly setting stream usage to kRecording.